react-frontend-app/src/App.js

```diff
@@ -13,61 +16,60 @@ import Display from './components/Display'
 import Footer from './components/Footer'
 import MainNavBar from './components/MainNavBar'
 
-class App extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {placeholder: "This is the default React message", input_val: ""}
-  }
+function App() {
+  const [surname, setSurname] = useState('');
+  const [zipcode, setZip] = useState('');
+  var prediction = {};
 
-  // Function that is called when the app first starts
-  componentDidMount() {
-    fetch('http://localhost:5000/hello').then(res => res.json()).then(data => {
-      this.setState({placeholder: data.result});
-    });
+  // Functions that are called whenever the text input is changed
+  function handleSurname(event) {
+    setSurname(event.target.value);
   }
 
-  // Function that is called whenever the text input is changed
-  handleChange(event) {
-    this.setState({input_val: event.target.value});
+  function handleZip(event) {
+    setZip(event.target.value);
   }
 
   // Function that called when the submit button is pressed
-  handleSubmit(event) {
+  function handleSubmit(event) {
     event.preventDefault();
-    console.log('Submit was pressed')
-  }
-
-  render() {
-    // In general, it's better to have a small return value with a lot of components that each live in their separate files
-    // As a way of building out architecture, we've built out a bunch of empty components that you can fill out as you build out the front-end.
-    // For any of these components, if you know of an existing library or component (ie: React bootstrap) that could work in place of one of the
-    // components below, feel free to substitute it in!
-    // This is just a skeleton. Feel free to get creative!
-
-    // It may be helpful to look at Bootstrap and React-Bootstrap as you build out these components. 
-    // Bootstrap: https://getbootstrap.com/ ; React-Bootstrap: https://react-bootstrap.github.io/
-    return (
-      <div className="App">
-        <p>Is flask working: {this.state.placeholder}</p>
-
-        <MainNavBar />
+    console.log('Submit was pressed');
+    console.log({surname}, {zipcode});
 
-        <Description />
+    axios.get(`http://localhost:5000/surgeo?surname=${surname}&zipcode=${zipcode}`)
+      .then(res => console.log(res.data));
 
-        <form onSubmit={this.handleSubmit}>
-            <label>
-              Insert Form Information:
-            <input type="text" value={this.state.input_val} onChange={this.handleChange}/>
-            </label>
-            <input type="submit" value="Submit" />
-        </form>
-
-        <Display />
+    // axios.get(`/surgeo?surname=${surname}&zipcode=${zipcode}`)
```

Comment:
remove comments when finished with everything

Reply:
unless they're descriptive/useful, but in this case seems like it's like old code